flusher.go

```diff
@@ -23,25 +23,25 @@ import (
 // Flush takes the slices of metrics, combines then and marshals them to json
 // for posting to Datadog.
 func (s *Server) Flush(interval time.Duration, metricLimit int) {
-	trace := trace.StartTrace("flush")
-	defer trace.Record("veneur.flush.trace", []*ssf.SSFTag{})
+	span := tracer.StartSpan("flush", trace.NameTag("veneur.flush.opentracing")).(*trace.Span)
+	defer span.Finish()
 
 	// right now we have only one destination plugin
 	// but eventually, this is where we would loop over our supported
 	// destinations
 	if s.IsLocal() {
-		s.FlushLocal(trace.Attach(context.Background()), interval, metricLimit)
+		s.FlushLocal(span.Attach(context.Background()), interval, metricLimit)
 	} else {
-		s.FlushGlobal(trace.Attach(context.Background()), interval, metricLimit)
+		s.FlushGlobal(span.Attach(context.Background()), interval, metricLimit)
 	}
 }
 
 func (s *Server) FlushGlobal(ctx context.Context, interval time.Duration, metricLimit int) {
-	trace := trace.SpanFromContext(ctx)
-	defer trace.Record("veneur.flush.FlushGlobal.trace", nil)
+	span, _ := trace.StartSpanFromContext(ctx, "flush", trace.NameTag("veneur.flush.FlushGlobal.opentracing"))
```

Comment:
From using the DD UI I often find it hard to read the name because they all end in `opentracing`. Could we move `opentracing` up so that each of these ends in the unique bit? Happy to discuss or whatever if you feel differently.